README.md

```diff
@@ -25,6 +25,12 @@ var Sentry = require('sentry-api').Client;
 // Create a new client using your Sentry DSN.
 var sentry = new Sentry('https://abc123:@app.getsentry.com');
 
+// If you're using authentication tokens, you'll want to pass the token as
+// a configuration option.
+var sentry = new Sentry('https://abc123:@app.getsentry.com', {
```

Comment:
this is the issue, in case i provide token, there is no dsn to provide. If token is provided, i guess you also need to provide the host if you are using a self hosted one.